### PR TITLE
DOCUMENTATIONS: Documentation update with custom serialization support

### PR DESCRIPTION
**Pull request by LBoullosa**
#154

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -211,7 +211,54 @@ var result = await PostFormAsync<FormUpload, ResultType>("https://example.com/up
 
 Note the linking of the FileName to the StreamContent via the name parameter in the attributes.
 
-### 4. Testing-Friendly Implementation
+### 4. Custom serialization/deserialization support
+RESTFulSense uses by default NewtonSoft serialization/deserialization support. However, in some scenarios this could present some drawbacks. Imagine
+1) your models don�t have NewtonSoft annotations (because your main project doesn�t uses NewtonSoft).
+1) you have some types which they need custom converters to specify how to deal with those types. For example, TimeOnly type. This type if you try serializate it with NewtonSoft or System.Text then raises an exception because it doesn�t know how to deal with it.
```
Comment (ElbekDeveloper) [inline readme.md:217]: image
Probably, you are using the character that my laptop doesn't understand try `'` or if you are using it split the word don't => do not

File: readme.md [hunk 2]
```diff
@@ -211,7 +211,54 @@ var result = await PostFormAsync<FormUpload, ResultType>("https://example.com/up
 
 Note the linking of the FileName to the StreamContent via the name parameter in the attributes.
 
-### 4. Testing-Friendly Implementation
+### 4. Custom serialization/deserialization support
+RESTFulSense uses by default NewtonSoft serialization/deserialization support. However, in some scenarios this could present some drawbacks. Imagine
```
Comment (ElbekDeveloper) [inline readme.md:215]: `:` after Imagine

**Review by ElbekDeveloper — CHANGES_REQUESTED** [review]

File: readme.md (hunk 1 above)
Comment (LBoullosa) [reply readme.md:217]: That is a file encoding issue, i don´t know why it was changed to ISO-8859 (Spanish)

File: readme.md [hunk 3]
```diff
@@ -211,7 +211,54 @@ var result = await PostFormAsync<FormUpload, ResultType>("https://example.com/up
...
+1) your models don�t have NewtonSoft annotations (because your main project doesn�t uses NewtonSoft).
```
[the remark this answers is not in the document]
Comment (LBoullosa) [reply readme.md:216]: ok!

**Pull request merged**